examples/file_upload_api/upload.py

```diff
@@ -16,20 +20,26 @@
 )
 from lexer_schemas.marketing_api.sms_event import SMSClick, SMSSend, SMSSubscribe
 from lexer_schemas.profile_api.customer_record import CustomerRecord
-from lexer_schemas.common import imported_api_names
 
 
 def upload_file(
     local_filename: str,
     record_type: str,
-    destination_filename: str,
+    destination_filename: Optional[str],
     destination_dataset_id: str,
     api_token: str,
 ) -> None:
 
     # A HTTP POST request to api.lexer.io/v1/uploads, if successful, will return a url that an api user
     # can run a PUT request on to upload a file. The destination parameters here allow lexer to automatically
     # trigger a dataset load after the upload of the file.
+
+    if destination_filename is None:
+        destination_filename = (
+            f"{destination_dataset_id}_{record_type}_{time():.0f}.ndjson"
```

Comment:
I think its worth mentioning this is the recommended file name format in the docs too

Reply:
The docs for this script, or the docs for the File Upload API, or both?

Reply:
I think the docs for the script at least, at some point we should probably mention it in Lexer Learn

Reply:
True, maybe there is a better place for it. It would be nice to have this documented as our preferred filename format somewhere other than code though, just so we can link to it

Reply:
I'll add it the the API docs at least.

Reply:
https://github.com/lexerdev/lexer-api/pull/69/commits/a59e3c47bd96f2d9d85ef085df240bfce0e068ab